### ISSUE TITLE
Musikcube will not play even tho it sees files.

### ISSUE THREAD

**Issue by Reaper176** [clangen/musikcube#689]
Platform: Arch linux
built from: source - latest
output driver: pipeware

no errors are shown but when i attempt to play a file(any) it simply shows playback is stopped just above the volume slider. I dont know what information might be helpful to you, so if you need anything let me know and i will do my best to provide it.
ty <3

**Comment by clangen** [comment]
Does it work if you try other output drivers? e.g. ALSA?

**Comment by Reaper176** [comment]
No, I have ran thru all the driver types. alsa, pipe, pulse, etc

**Comment by clangen** [comment]
My guess is that the audio decoder is failing. Recently ffmpeg 7.0 was released, which is not compatible with ffmpeg 6.0. My guess is you're either using a version of musikcube that's expecting 6.0 and you have 7.0, or vice versa.

If you press the backtick key "`" you can get to the console and sometimes there's some additional info in there.

**Comment by Reaper176** [comment]
libffmpegdecoder.so error shown with `
I take it that means the ffmpeg thing. 
should i downgrade or is there a fix coming?

**Comment by elpres** [comment]
(Disclaimer: new user, just installed Musikcube yesterday for the first time). I've a similar problem with the newest build available on Arch from AUR, but it only affects some files. Playing these same files with `ffplay` works with no problems. If the player produces any error messages, where would I look for them? There isn't anything in the journal. Would you like to have one such file for testing?

**Comment by ru55-t** [comment]
I've also had this issue on a couple of occasions and yeah, it's usually an ffmpeg version issue but I've got it working again after rebuilding the aur package (Endeavouros) so YMMV.
Pressing ~ (tilde by default) will show the debug log which should give you an error message.
Downgrading can be problematic so do so as a last resort, caveat emptor.
